src/ralph/models/edx/server.py

```diff
@@ -33,6 +35,11 @@ class ServerEventModel(BaseEventModel):
                 The JSON string is truncated at 512 characters resulting in invalid JSON.
     """
 
+    __selector__ = (
+        Rule(ModelField("event_source"), "server"),
+        Rule(ModelField("event_type"), ModelField("context__path")),
```

Comment:
From an API user perspective, I would love to write rules as ORM-like queries:

```suggestion
        Rule(event_type=LazyModelField("context__path")),
```

We'll discuss this IRL, but you got the idea.

Reply:
This looks interesting) Thanks) I will have a look how to make this work)

Reply:
Updated the PR with this change) But I kept the optional `_operation` argument to be able to use other operations than equality for the checks (hope no field will be named `_operation` in the future models).